.github/workflows/clabot.yml

```diff
@@ -0,0 +1,21 @@
+name: "CLA Signature Bot"
+on:
+  issue_comment:
+    types: [created]
+  pull_request:
+    types: [opened,closed,synchronize]
+
+jobs:
+  clabot:
+    runs-on: ubuntu-latest
+    steps:
+    - name: "CLA Signature Bot"
+      uses: roblox/cla-signature-bot@v2.0.0
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        whitelist: "LPGhatguy,ZoteTheMighty,cliffchapmanrbx"
```

Comment:
Could you add me to the whitelist for any lua frameworks like Roact, Rodux, etc.

Reply:
please add me to the whitelists as well. thanks!